README.md

```diff
@@ -79,9 +79,33 @@ In graph theory terminology, they are `leaf` nodes of your graph.
 
 Run `utensor-cli convert --help` for detailed information.
 
-# Example
+# Use `utensor_cgen` as Library
+
+## Subgraph Isomorphic Matcher and Nodes Fusion
+
+With `uTensorGraphMatcher`, performing a subgraph isomorphic match, replace or manipulate the matched subgraph is just a few line of code!
+
+### Node Fusion
```

Comment:
For these few examples, you might want one or two sentences explaining what they user is looking for.

Reply:
Right. I'll elaborate it more.